src/assignment/pkg9/DiceReaderTest.java

```diff
@@ -5,18 +5,63 @@
  */
 package assignment.pkg9;
 
+import java.util.ArrayList;
+import java.util.List;
+import dice.DiceTower;
+
 /**
  *
  * @author Paul Scarrone
  */
-
 // This test passes now and gives an example of how your file should work
 // Once you are done with reading and calculating dice you will need to update
 // this test to pass again
 public class DiceReaderTest {
-  public static void test_reader(){
-	DiceFileReader diceReader = new DiceFileReader("dice.txt");
-	diceReader.read();
-	System.out.println(diceReader.getLines().toString().equals("[]"));
+
+  public static void test_reader() {
+    DiceFileReader diceReader = new DiceFileReader("dice.txt");
```

Comment:
I always have to do a `./` infront of my local files so `"./dice.txt"`

but if this works ignore me